content/pt-br/docs/concepts/windows/intro.md

```diff
@@ -0,0 +1,286 @@
+---
+title :Contêineres do Windows no Kubernetes
+content_type: concept
+weight: 65
+---
+
+<!-- overview -->
+
+Os aplicativos do Windows constituem uma grande parte dos serviços e aplicativos executados em muitas organizações. [Windows containers](https://aka.ms/windowscontainers)
+fornecem uma maneira de encapsular processos e dependências de pacotes, facilitando o uso de práticas de DevOps e seguindo padrões nativos de nuvem para aplicativos do Windows.
+
+As organizações com investimentos em aplicativos baseados em Windows e aplicativos baseados em Linux não precisam procurar orquestradores separados para gerenciar suas cargas de trabalho, levando a um aumento eficiências operacionais em suas implantações, independentemente do sistema operacional.
+
+<!-- body -->
+
+## Windows nodes em Kubernetes
+
+Para habilitar a orquestração de contêineres do Windows no Kubernetes, inclua nós do Windows em seu cluster Linux existente. Agendamento de contêineres do Windows em {{< glossary_tooltip text="Pods" term_id="pod" >}} em O Kubernetes é semelhante ao agendamento de contêineres baseados em Linux.
```

Comment:
Some suggestions for this paragraph:
* I'd write "nós Windows" instead of "nós do Windows", as the latter seems to mean that the node belongs to Windows, but in reality it just runs Windows. Same applies to "contêineres do Windows".
* How about writing "no Kubernetes" instead of "em O Kubernetes"?